pkg/kubectl/cmd/util/helpers.go

```diff
@@ -204,25 +206,28 @@ func StandardErrorMessage(err error) (string, bool) {
 }
 
 // MultilineError returns a string representing an error that splits sub errors into their own
-// lines. The returned string will end with a newline.
+// lines. The returned string will end with a newline. The prefix should not end in . or : or \n.
 func MultilineError(prefix string, err error) string {
+	if err == nil {
+		return fmt.Sprintf("%s.\n", prefix)
```

Comment:
In fact, what the function actually means is this:

```
func multilineErrorString(err string, subErr error) string {
```

Then the check for nil actually makes sense and it's fine to print the `err` when subErr is nil.